test/src/nyc-test.js

```diff
@@ -161,7 +161,7 @@ describe('nyc', function () {
     })
 
     describe('custom require hooks are installed', function () {
-      it('wraps modules with coverage counters when the custom require hook compiles them', function () {
+      /* it('wraps modules with coverage counters when the custom require hook compiles them', function () {
```

Comment:
@novemberborn - we still have a few tests that test `index.js`, including this one. I believe you had a PR that fixed some of these.